src/main.lua

```diff
@@ -1775,13 +1765,26 @@ end, function(msg)
 
 	print("Ticking from " .. lastCreatedEpochIndex .. " to " .. targetCurrentEpochIndex)
 
+	-- tick the demand factor
+	local demandFactor = demand.updateDemandFactor(msg.Timestamp)
+	if demandFactor ~= nil then
+		table.insert(newDemandFactors, demandFactor)
+		Send(msg, {
+			Target = msg.From,
+			Action = "Demand-Factor-Updated-Notice",
+			Data = tostring(demandFactor),
+		})
+	end
```

Comment:
demand factor updates are actually separate from epochs. So they can be pulled up and send their own updates.